src/Uno.UI/UI/Xaml/Media/AcrylicBrush/Android/CustomRealtimeBlurView.cs

```diff
@@ -0,0 +1,431 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Android.App;
+using Android.Content;
+using Android.Graphics;
+using Android.OS;
+using Android.Runtime;
+using Android.Util;
+using Android.Views;
+using Uno.Extensions;
+using Uno.Logging;
+using Uno.UI.Xaml.Media;
+
+namespace Uno.UI.Xaml.Media
+{
+	internal partial class RealtimeBlurView : View
+	{
+		private float mDownsampleFactor = 4;
+		private int mOverlayColor; // default #aaffffff
+		private float mBlurRadius = 10; // (0 < r <= 25)
+
+		private IBlurImpl mBlurImpl;
+		private bool mDirty;
+		private Bitmap mBitmapToBlur, mBlurredBitmap;
+		private Canvas mBlurringCanvas;
+		private bool mIsRendering;
+		private Paint mPaint;
+		private readonly Rect mRectSrc = new Rect();
+		private readonly Rect mRectDst = new Rect();
+		// mDecorView should be the root view of the activity (even if you are on a different window like a dialog)
+		private View mDecorView;
+		// If the view is on different root view (usually means we are on a PopupWindow),
+		// we need to manually call invalidate() in onPreDraw(), otherwise we will not be able to see the changes
+		private bool mDifferentRoot;
+		private static int RENDERING_COUNT;
+		private static int BLUR_IMPL;
+
+		private readonly PreDrawListener _preDrawListener;
+
+		public RealtimeBlurView(Context context) : base(context)
+		{
+			_preDrawListener = new PreDrawListener(this);
+			mBlurImpl = GetBlurImpl(); // provide your own by override getBlurImpl()			
+			mBlurRadius = TypedValue.ApplyDimension(ComplexUnitType.Dip, 10, context.Resources.DisplayMetrics);
+			//mDownsampleFactor = a.getFloat(R.styleable.RealtimeBlurView_realtimeDownsampleFactor, 4);
+
+			mPaint = new Paint();
+		}
+
+		public void SetOverlayColor(int color, bool invalidate = true)
```

Comment:
Codacy found an issue: [This method signature overlaps the one defined on line 206, the default parameter value can't be used.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5932087)